guide/14-deep-learning/fairness_in_tabulardata_modeling_using_mlmodel.ipynb

```diff
@@ -0,0 +1,1087 @@
+{
```

Comment:
<p>We can see that except for the large RMSE difference, the report before mitigation suggests that the base model is already quite fair with respect to gender. While the fairness flags being True confirms that the model meets the fairness standards for gender, the large RMSE difference seems to be an issue. Let us see if applying a suitable mitigation strategy can improve it. First, we will define the fairness arguments as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2279/discussion/#comment-2461638495'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>